src/store/localStorage.ts

```diff
@@ -0,0 +1,23 @@
+import { Cart } from '../types';
+
+export const loadCart = () => {
+  try {
+    const serializedCart = localStorage.getItem('cart');
+    if (serializedCart === null) return {};
+
+    const cart = JSON.parse(serializedCart);
+    return cart as Cart;
+  } catch (err) {
+    return {};
```

Comment:
If we console.log for store, shouldn't' we do the same for load? And perhaps it should be a `notify` so that the user sees that where was an error and it's not a mystery why something didn't save/load.

Reply:
Good catch, I was planning on removing the log and doing what `loadCart` does. But I like the idea of adding a `notify` so something like:
```tsx
try {
// ...
} catch (err) {
	notify(err);
	return {};
}
// ...
```
Does that sound good?